src/pyarcrest/arc.py

```diff
@@ -242,13 +243,15 @@ def newDelegation(self, token=None, proxypath=None):
             if not token:
                 token = self.token
             headers["X-Token-Delegation"] = f"Bearer {token}"
+            headers["Authorization"] = f"Bearer {token}"
             params["type"] = "jwt"
 
+        endpoint = f"{self.apiPath}/delegations"
         resp = self._request("POST", "/delegations", headers=headers, params=params, token=token)
         respstr = resp.read().decode()
 
         if token:
-            if resp.status != 200:
+            if resp.status != 201:
```

Comment:
If 201 is returned for tokens, we can again report the fix for the [this](https://www.nordugrid.org/arc/arc7/tech/rest/rest.html#jwt-delegation) section of documentation. You can also submit a separate pull request for pyarcrest to fix this.